lib/css_parser/rule_set.rb

```diff
@@ -22,6 +22,30 @@ class RuleSet
       ['border-width', %w[border-top-width border-right-width border-bottom-width border-left-width]]
     ].freeze
 
+    class DeclarationValue
+      attr_reader :value
+      attr_accessor :important, :order
+
+      def initialize(value, important: nil, order: 0)
+        self.value = value
+        @important = important.nil? ? !value.match(CssParser::IMPORTANT_IN_PROPERTY_RX).nil? : important
+        @order = order
+      end
+
+      def value=(value)
+        value = value.to_s.sub(/\s*;\s*\Z/, '').gsub(CssParser::IMPORTANT_IN_PROPERTY_RX, '').strip
+        raise ArgumentError, 'value is empty' if value.empty?
+
+        @value = value.freeze
+      end
+
+      def to_s
+        return value unless important
+
+        "#{value} !important"
+      end
```

Comment:
`def ==(other)`
>do we really need this ?
>default == should do roughly the same

Yeah, minitest complained that it can't compare values and I added this. Somehow after removal tests still passing, so /shrug.

Reply:
I think this came from a assert_equal, maybe that is gone now 🤷